src/librbd/mirror/DisableRequest.cc

```diff
@@ -420,6 +423,8 @@ Context *DisableRequest<I>::handle_unregister_client(
 
   if (m_error_result < 0) {
     *result = m_error_result;
+    locker.unlock();
+
     return m_on_finish;
   }
 
```

Comment:
@dillaman I meant `locker.unlock()` after `send_get_clients()`.

Reply:
sorry, before (not after).

Reply:
Done